grafana/grafana/base/jupyterhub-sli-dashboard.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: odh-jupyterhub-sli
+  labels:
+    app: grafana
+spec:
+  url: https://raw.githubusercontent.com/odh-manifests/grafana/grafana/base/dashboards/jupyterhub-sli-slo.json
```

Comment:
I agree on linking to a static sha or tag when loading a json object from a url.  In this instance, I don't think we need the `url` since this json would be hosted in `odh-manifests`.